Applications/Spire/Source/Charting/ChartView.cpp

```diff
@@ -771,6 +769,7 @@ void ChartView::on_left_mouse_button_press(const QPoint& pos) {
 
 void ChartView::on_left_mouse_button_release() {
   if(m_draw_state == DrawState::LINE || m_draw_state == DrawState::POINT) {
+    m_line_mouse_offset.reset();
```

Comment:
Prefer ```m_line_mouse_offset = std::nullopt;```

Reply:
What's the difference? I also just read that I can assign it an empty braced initializer list.

Reply:
Only difference is coding style. For whatever reason the C++ community has converged to this convention.